c172-checklists.xml

```diff
@@ -2739,9 +2760,15 @@
                 <scale>2.7686</scale>
             </marker>
             <condition>
-                <property>sim/model/c172p/securing/tiedownL-visible</property>
+                <or>
+                    <property>sim/model/c172p/securing/pitot-cover-visible</property>
```

Comment:
do not use the pitot tube visibility for anything other than the pitot. Right now you applied it to all securing items.

Reply:
Sorry - copied that in meaning to change it for each item and forgot all about it!